easybuild/easyconfigs/x/xmitgcm/xmitgcm-0.5.2-foss-2022a.eb

```diff
@@ -0,0 +1,41 @@
+# Author: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'PythonBundle'
+
+name = 'xmitgcm'
+version = '0.5.2'
+
+homepage = 'https://github.com/MITgcm/xmitgcm'
+description = """xmitgcm is a python package for reading MITgcm binary 
+MDS files into xarray data structures. By storing data in dask arrays, 
+xmitgcm enables parallel, out-of-core analysis of MITgcm output data."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('MITgcmutils', '0.1.2'),
+    ('dask', '2022.10.0'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('cachetools', '5.2.0', {
+        'checksums': ['6a94c6402995a99c3970cc7e4884bb60b4a8639938157eeed436098bf9831757'],
+    }),
+    ('xarray', '2022.9.0', {
```

Comment:
Please use a dependency on the easyconfig of `xarray`

Reply:
Done. Thanks for finding that, I frankly forgot about it is in there as well.

Reply:
Thanks, I had to downgrade the version of `xarray` because there are already other easyconfigs depending on it this toolchain. Test should be fine now.